public/static/locales/es/donate.json

```diff
@@ -61,7 +60,6 @@
   "titleToShare": "Plantando árboles contra la crisis climática!",
   "giftToName": "Regalo a",
   "plantTreesWith": "Plantar árboles con",
-  "textToShare": "Únete a {{name}} y miles de otros para plantar un trillón de árboles para hacer frente a la crisis climática #Dejar de hablar, empezar a plantar... TrillionTrees @trilliontrees",
```

Comment:
We seem to have lost some Spanish versions as unapproved resources do no get exported!

Reply:
confirmed this resource